src/Form/AsuBrandSettingsForm.php

```diff
@@ -91,6 +91,21 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         container with that new ID will be added to your site's HTML. This
         setting is independent of whether the ASU Universal GTM is enabled."),
     ];
+    //Search settings
+    $form['asu_brand']['search_config'] = array(
+      '#type' => 'fieldset',
+      '#title' => $this->t('Search Settings'),
+    );
+    $form['asu_brand']['search_config']['asu_brand_search_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Search URL'),
+      '#default_value' => $config->get('asu_brand.asu_brand_search_url'),
```

Comment:
Let's add a description: `URL to use for search in the header. Please don't update unless instructed. Format should match: https://search.asu.edu/search`